docs/source/advanced/advanced_gpu.rst

```diff
@@ -1,13 +1,15 @@
+.. _model_parallel:
```

Comment:
I'd consider renaming this, as many of these are data parallel techniques. we're not addressing tensor parallelism or pipeline parallelism here

Reply:
I think this doc contains info regarding sharding/deepspeed mostly, so this is model parallelism, right?

Reply:
We're not addressing TP or PP yet :) I hope we will eventually in here once a general solution becomes readily available.